services/core-web/common/actionCreators/minespaceActionCreator.js

```diff
@@ -63,3 +63,86 @@ export const deleteMinespaceUser = (minespaceUserId) => (dispatch) => {
     .catch(() => dispatch(error(reducerTypes.DELETE_MINESPACE_USER)))
     .finally(() => dispatch(hideLoading()));
 };
+
+// MineSpace EMLI contact
+export const fetchEMLIContacts = () => (dispatch) => {
+  dispatch(showLoading());
+  dispatch(request(reducerTypes.GET_EMLI_CONTACTS));
+  return CustomAxios()
+    .get(ENVIRONMENT.apiUrl + API.EMLI_CONTACTS, createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_EMLI_CONTACTS));
+      dispatch(minespaceActions.storeEMLIContacts(response.data));
+      return response;
+    })
+    .catch(() => dispatch(error(reducerTypes.GET_EMLI_CONTACTS)))
+    .finally(() => dispatch(hideLoading()));
+};
+
+export const fetchEMLIContactsByRegion = (region, isMajorMine) => (dispatch) => {
```

Comment:
is this only for region or major mine as well?

Reply:
MM as well - given the region and the MM flag, the API will return either the regional contacts based on region or the MM contacts based on region (and the standard MM contacts that apply to every region)